app/controllers/versions_controller.rb

```diff
@@ -0,0 +1,5 @@
+class VersionsController < ApplicationController
+  def index
+    @versions = PaperTrail::Version.where(item_id: params.require(:item_id), item_type: params.require(:item_type))
```

Comment:
thinking more about this, how confident are we that PaperTrail won't ever `const_get(item_type)`? Do we need a whitelist?